src/test/unit/test_lumigo_utils.py

```diff
@@ -247,12 +248,50 @@ def func():
             f'{{"{SKIP_SCRUBBING_KEYS[0]}": {{"password": 1}}}}',
         ),
         ([{"password": 1}, {"a": "b"}], '[{"password": "****"}, {"a": "b"}]'),  # list of dicts
+        (  # Dict of long list
+            {"a": [{"key": "value", "password": "value", "b": "c"}]},
+            '{"a": [{"key": "****", "password": "****", "b": "c"}]}',
+        ),
+        (  # Multiple nested lists
+            {"a": [[[{"c": [{"key": "v"}]}], [{"c": [{"key": "v"}]}]]]},
+            '{"a": [[[{"c": [{"key": "****"}]}], [{"c": [{"key": "****"}]}]]]}',
+        ),
     ],
 )
 def test_lumigo_dumps(value, output):
     assert lumigo_dumps(value, max_size=100) == output
 
 
+@pytest.mark.parametrize(
+    ("value", "omit_skip_path", "output"),
+    [
+        ({"a": "b", "Key": "v"}, ["Key"], '{"a": "b", "Key": "v"}'),  # Not nested
+        (  # Nested with list
+            {"R": [{"o": {"key": "value"}}]},
+            ["R", "o", "key"],
+            '{"R": [{"o": {"key": "value"}}]}',
+        ),
+        (  # Doesnt affect other paths
+            {"a": {"key": "v"}, "b": {"key": "v"}},
+            ["a", "key"],
+            '{"a": {"key": "v"}, "b": {"key": "****"}}',
+        ),
+        (  # Nested items not affected
+            {"key": {"password": "v"}},
```

Comment:
this is surprising!
I really thought that this is part of the logic we want- think about putting a nested key on ddb. IMO, we don't want to scrub it...

Reply:
But if someones saves passwords in DDB, we want to scrub the passwords, isn't it? 🤔

Reply:
but this is inside the `Key` key, that shouldn't be scrubbed 🤔

Reply:
We just don't want scrub the immediate value of `Key` because `Key` is part of the AWS api.
We want to keep scrubbing the data of the client but not the AWS api.

Reply:
I would expect that it will be: `'{"key": "****"}'`